lib/devicetrust/native/device_darwin.h

```diff
@@ -44,6 +44,13 @@ int32_t DeviceKeySign(Digest digest, Signature *sigOut);
 // DeviceData contains collected data for the device in use.
 typedef struct _DeviceData {
   const char *serial_number;
+  const char *model;
```

Comment:
Nit: I'd appreciate an example of what "model" looks like. Is this a model number like "A2485" or is this something else ?

Reply:
Done. There are examples in the collected data proto for most fields as well.